build-packages.sh

```diff
@@ -29,10 +29,11 @@ case $OSName in
 
     Linux)
         source /etc/os-release
+        VersionMajor=$(echo $VERSION_ID | awk 'match($0, /[0-9]+/) { print substr($0, RSTART, RLENGTH) }')
```

Comment:
It looks like we switched from debian.8.2 -> debian.8 about 3 weeks ago.  I'm not certain if this is the long-term plan for debian and rhel (to just package with the version major).  If this is likely to change, then this code change (somewhat of a hack), could become an annoyance. Where does VERSION_ID come from?

/cc @ellismg